src/Util/FlatpickrUtil.php

```diff
@@ -147,7 +146,7 @@ protected function addFrontendAssets(array $attributes): void
             return;
         }
 
-        if (!$this->dcaUtil->getOverridableProperty('useFlatpickrForDatepicker', $this->getPageWithParents())) {
+        if (!empty($this->dcaUtil->getOverridableProperty('useFlatpickrForDatepicker', $this->getPageWithParents()))) {
```

Comment:
Why empty? this method delivers a char (as useFlatpickrForDatepicker is defined), i.e. a "boolean". So the `!`-check has been ok.